tools/syz-prog2c/prog2c.go

```diff
@@ -58,23 +58,25 @@ func main() {
 		os.Exit(1)
 	}
 	opts := csource.Options{
-		Threaded:      *flagThreaded,
-		Collide:       *flagCollide,
-		Repeat:        *flagRepeat != 1,
-		RepeatTimes:   *flagRepeat,
-		Procs:         *flagProcs,
-		Sandbox:       *flagSandbox,
-		Fault:         *flagFaultCall >= 0,
-		FaultCall:     *flagFaultCall,
-		FaultNth:      *flagFaultNth,
-		EnableTun:     *flagEnableTun,
-		UseTmpDir:     *flagUseTmpDir,
-		EnableCgroups: *flagCgroups,
-		EnableNetdev:  *flagNetdev,
-		ResetNet:      *flagResetNet,
-		HandleSegv:    *flagHandleSegv,
-		Repro:         false,
-		Trace:         *flagTrace,
+		Threaded:            *flagThreaded,
+		Collide:             *flagCollide,
+		Repeat:              *flagRepeat != 1,
+		RepeatTimes:         *flagRepeat,
+		Procs:               *flagProcs,
+		Sandbox:             *flagSandbox,
+		Fault:               *flagFaultCall >= 0,
+		FaultCall:           *flagFaultCall,
+		FaultNth:            *flagFaultNth,
+		EnableTun:           *flagEnableTun,
+		UseTmpDir:           *flagUseTmpDir,
+		EnableCgroups:       *flagCgroups,
+		EnableNetdevNames:   *flagNetdev,
```

Comment:
We need 3 separate flags here, otherwise it won't be possible to generate C repro the same way manager can, and it won't be possible to manually test the functionality and manually minimize C repro.
We generally just have 1-to-1 correspondence between options and flags here.